src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -1163,7 +1166,8 @@ constexpr std::array _objectTypeNames = {
     "Footpath Surface",
     "Footpath Railings",
     "Audio",
-    "Guest Names",
+    "Peep Names",
+    "Peep Animations",
```

Comment:
It doesn’t need to be changed for this PR, but this should just use the translated names that the Object Selection uses. (If we even keep the console.)

Reply:
Indeed, the whole thing is pretty much not translated right now. This could be changed to map to string ids with relative ease, though. Perhaps this could be a 'good first issue'?